kedro-org/vscode-kedro
# Release 0.2.1 (official release)

# How to install
fix #88 

you can install pre-release version of extension from marketplace directly:
- Click the arrow next to the `Install` button
<img width="585" alt="image" src="https://github.com/user-attachments/assets/39c8594e-6265-4538-9ffa-5eea711d6290">

# How to test
- Make sure you open VSCode at the root of a Kedro project
- Make sure VSCode is using the correct virtual environment, i.e. the environment that you can do `kedro run`
- Bonus: Start a fresh new environment, kedro project to test because we need to make sure dependencies is not a problem. We added a new flow in case user are missing certain dependencies.
- Test Kedro: Run Viz
- playaround with namespace, non spaceflights project structure.

- #71
